lib/manager.rb

```diff
@@ -0,0 +1,93 @@
+require 'time'
+require 'date'
+require_relative 'room'
+require_relative 'reservation'
+
+
+module Hotel
+  class Manager
+    attr_reader :num, :all_rooms, :rm_reservations, :recloc
```

Comment:
This should only define readers for instance variable of the `Hotel` class. The only of these 4 that is an instance variable of `Hotel` is `all_rooms`. 
```suggestion
    attr_reader :all_rooms
```

Reply:
I imagine you might understand that and just forgot to update this line.